core/mutator.ts

```diff
@@ -236,65 +236,63 @@ export class Mutator extends Icon {
   override updateEditable() {
     super.updateEditable();
     if (!this.getBlock().isInFlyout) {
-      if (this.getBlock().isEditable()) {
-        if (this.iconGroup_) {
-          dom.removeClass(this.iconGroup_, 'blocklyIconGroupReadonly');
-        }
-      } else {
+      if (!this.getBlock().isEditable) {
         // Close any mutator bubble.  Icon is not clickable.
         this.setVisible(false);
-        if (this.iconGroup_) {
-          dom.addClass(this.iconGroup_, 'blocklyIconGroupReadonly');
-        }
+      }
+      if (this.iconGroup_) {
+        dom.removeClass(this.iconGroup_, 'blocklyIconGroupReadonly');
       }
     }
   }
 
   /** Resize the bubble to match the size of the workspace. */
-  private resizeBubble_() {
+  private resizeBubble() {
+    // If the bubble exists, the workspace also exists.
+    const ws = this.workspace as WorkspaceSvg;
```

Comment:
I like using the `!` instead of casting, the intention is more clear (with the cast, i scrolled back up to see if this was allowed to be a plain `Workspace` to understand why we needed the cast). does it work to `const ws = this.workspace!;`?

Alternatively I think `if (!this.workspace) return;` is even better since it's type safer and about the same amount of code. you could do the same below in `setVisible` to eliminate more non-null assertions